lib/omise/res/OmiseApiResource.php

```diff
@@ -203,7 +203,15 @@ private function _executeTest($url, $requestMethod, $key, $params = null)
         $request_url = preg_replace('#^(http|https)://#', '', $url);
 
         // Remove slash if it had in last letter.
-        $request_url = rtrim($request_url, '/');
+        $parsed = parse_url($request_url);
```

Comment:
@iwat Saw that we can also remove the above line 
```php
$request_url = preg_replace('#^(http|https)://#', '', $url);
```
and use `$parsed['host']` from the parsed component at `$parsed = parse_url($url);` instead.